src/services/chatroom.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Chatroom } from '@/entities/nosql';
+import { HttpException } from '@/exceptions';
+import { model } from 'mongoose';
+
+class ChatroomService {
+  public async createChatroom(userId: number, receiverId: number): Promise<void> {
+    const chatroom = model('Chatroom', Chatroom);
+    const createChat = new chatroom({
+      isGroup: true,
```

Comment:
should this always be true? if it's just a one to one chatroom it's not really a group